dell/xps/13-7390/default.nix

```
@@ -0,0 +1,10 @@
+{ lib, pkgs, ... }:
+
+{
+  imports = [
+    ../../../common/cpu/intel
+    ../../../common/pc/laptop
```

I think we can also enable the SSD module for trim support.

```suggestion
    ../../../common/pc/laptop
    ../../../common/pc/laptop/ssd
```

---

As it comes with NVME by default.